libsrc/hyperion/GrabberWrapper.cpp

```diff
@@ -56,6 +56,8 @@ GrabberWrapper::GrabberWrapper(const QString& grabberName, Grabber * ggrabber, i
 
 GrabberWrapper::~GrabberWrapper()
 {
+	_timer->stop();
+	delete _timer;
```

Comment:
:thought_balloon: Should such an operation be performed by [a smart pointer](https://doc.qt.io/qt-6/qscopedpointer.html "Using a QScopedPointer class?")?

Reply:
Yes. That would work.

Unfortunately, we are only two developers currently and do have the bandwidth cleaning up the whole code base proactively now. Esp. Like for the sample where the code correctly works and there is limited immediate benefit.

Reply:
How do you think about to avoid redundant code a bit more? :thinking:

Reply:
I am not sure, if I get your question…
In case of new additions or refactoring code,  now Smartpointers can be used to avoid additional boilerplate and make code more maintainable.
Nevertheless, there are more serious problems out there than removing working statements like above.
E.g. the grabber design is not consistent any longer and not every grabber is scalable.
Spending refactoring effort in that light would make great sense including removing redundant code.

We would be happy, if you and others would give us a helping hand in cleaning up current code and/or designs.